### PR TITLE
Yml settings files organized by directory

### PR DESCRIPTION
**Pull request by tm-luke**
changes to .ps1 files to reflect generic names of yml files.

### CODE REVIEW

File: Cloudmarque.Azure/Public/Common/Get-CmAzSubscriptionName.ps1 [hunk 1]
```diff
@@ -1,4 +1,17 @@
 function Get-CmAzSubscriptionName {
+	<#
+		.Synopsis
```
Comment (tm-luke) [inline Cloudmarque.Azure/Public/Common/Get-CmAzSubscriptionName.ps1:3]: Going to be a private method, documentation wont be needed.

File: Cloudmarque.Azure/Public/Core/recoveryvault/New-CmAzRecoveryServicesVault.ps1 [hunk 2]
```diff
@@ -22,19 +22,19 @@ function New-CmAzRecoveryServicesVault {
          Core
 
         .Example
-         New-CmAzRecoveryServicesVault -SettingsFile C:\ProjectDirectory\RecoveryServicesVault.yml
+         New-CmAzRecoveryServicesVault -SettingsFile <FilePath>/<SettingsFileName>.yml
 
         .Example
-         New-CmAzRecoveryServicesVault -SettingsFile C:\ProjectDirectory\RecoveryServicesVault.yml -PolicySettingsFile C:\ProjectDirectory\RecoveryServicesPolicy.yml
+         New-CmAzRecoveryServicesVault -SettingsFile <FilePath>/<SettingsFileName>.yml -PolicySettingsFile <FilePath>/<PolicySettingsFileName>.yml
```
Comment (tm-luke) [inline Cloudmarque.Azure/Public/Core/recoveryvault/New-CmAzRecoveryServicesVault.ps1:28]: Is this on the same line in code?

File: Cloudmarque.Azure/Resources/Project/naming.yaml [hunk 3]
```diff
@@ -1,54 +0,0 @@
-tokens:
```
Comment (tm-luke) [inline Cloudmarque.Azure/Resources/Project/naming.yaml:1]: Just double checking, did the name generator work without issue?

Comment (tmshaunporter) [reply Cloudmarque.Azure/Resources/Project/naming.yaml:1]: My basic checks seemed fine

File: Cloudmarque.Azure/Public/Common/Get-CmAzSubscriptionName.ps1 (hunk 1 above)
Comment (tmshaunporter) [reply Cloudmarque.Azure/Public/Common/Get-CmAzSubscriptionName.ps1:3]: Sorry, removed now.

File: Cloudmarque.Azure/Public/Core/recoveryvault/New-CmAzRecoveryServicesVault.ps1 (hunk 2 above)
Comment (tmshaunporter) [reply Cloudmarque.Azure/Public/Core/recoveryvault/New-CmAzRecoveryServicesVault.ps1:28]: Seems correct.

**Pull request closed without merging**